packages/rancher-alerting-drivers/charts/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+Rancher-alerting-drivers is now installed on the cluster!
+Please check the sub-chart's documentation for configuration and usage.
```

Comment:
I like this. Wondering if we should expand Logging v2's one to look similar to this.